packages/terra-framework-docs/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ## 1.43.0 - (October 25, 2023)
 
 * Added
+  * Added examples and test to `terra-table` for row selection mode.
```

Comment:
Can you move this to the released section?

Also looks like these entries for `packages/terra-framework-docs/CHANGELOG.md` also got merged in the wrong section:

https://github.com/cerner/terra-framework/pull/1858